cogs/utils/settings.py

```diff
@@ -171,6 +175,26 @@ def set_server_mod(self, server, value):
         self.bot_settings[server.id]["MOD_ROLE"] = value
         self.save_settings()
 
+    def get_server_prefixes(self, server):
+        if server is None or server.id not in self.bot_settings:
+            return self.prefixes
+        p = self.bot_settings[server.id].get("PREFIXES")
```

Comment:
I wanted to make it return `None` in case: `PREFIXES` is absent or it's set to `None` or it's `[]`